src/components/accessories/radiology/series/Series.tsx

```diff
@@ -0,0 +1,247 @@
+        operators: series.series?.operatorsName ?? "",
+        protocol: series.series?.protocolName ?? "",
+        station: series.series?.stationName ?? "",
+        status: series.status ?? "",
+      };
+    });
+  };
+
+  const navigateToStudies = useCallback(() => {
+    navigate("..");
+  }, [navigate]);
+
+  useEffect(() => {
+    return () => {
+      getStudySeriesReset();
+      if (previewState.status !== "IDLE") {
+        getInstancePreviewReset();
+      }
+    };
+  }, [dispatch]);
+
+  const handlePreview = useCallback(
+    (row: any) => () => {
+      dispatch(getInstancePreview(row.id));
+    },
+    [dispatch]
+  );
+
+  const handleClosePreview = useCallback(() => {
+    setOpenPreview(false);
+    dispatch(getInstancePreviewReset());
+  }, [dispatch, setOpenPreview]);
+
+  useEffect(() => {
+    if (previewState.hasSucceeded) {
+      setOpenPreview(true);
+    }
+  }, [previewState.status, setOpenPreview]);
+
+  const handleViewSeries = useViewMore("series");
+
+  return (
+    <div className="series">
+      {(() => {
+        switch (seriesState.status) {
+          case "FAIL":
+            return (
+              <InfoBox
+                type="error"
+                message={t(
+                  seriesState.error?.message ?? "common.somethingwrong"
+                )}
+              />
+            );
+          case "LOADING":
+            return (
+              <CircularProgress
+                style={{ marginLeft: "50%", position: "relative" }}
+              />
+            );
+
+          case "SUCCESS":
+            return (
+              <>
+                <Button
+                  variant="text"
+                  color="secondary"
+                  onClick={navigateToStudies}
+                >
+                  <ChevronLeft /> {t("radiology.series.backToStudies")}
+                </Button>
+                {study?.title && (
+                  <p className="series__StudyTitle">
+                    {study.title} {study.date && " | " + study.date}
+                  </p>
+                )}
+                {previewState.hasFailed && (
+                  <InfoBox
+                    type="error"
+                    message={t(
+                      previewState.error?.message ?? "common.somethingwrong"
+                    )}
+                  />
+                )}
+                <Table
+                  rowData={formatDataToDisplay(seriesState.data ?? [])}
+                  dateFields={dateFields}
+                  tableHeader={header}
+                  labelData={label}
+                  columnsOrder={order}
+                  rowsPerPage={5}
+                  isCollapsabile={true}
+                  renderCustomActions={(row) => (
+                    <div className="series__actions">
+                      <Tooltip title={t("radiology.series.viewMore")}>
+                        <IconButton onClick={handleViewSeries(row)}>
+                          <Visibility />
```

Comment:
Is it possible to use a more... contextual icon? Like the one illustration an external link? Or something similar?